pep-0664.rst

```diff
@@ -81,7 +81,15 @@ Features for 3.11
 
 Some of the notable features of Python 3.11 include:
 
- ** Watch this space :) **
+* A ~1.25x speedup from 3.10
+* New -P command line option and PYTHONSAFEPATH environment variable.
+* :pep:`654`, Exception Groups and except*.
+* :pep:`657`, Enhanced error locations in tracebacks.
+* :pep:`646`, Variadic generics.
+* :pep:`655`, Marking individual TypedDict items as required or potentially-missing.
+* :pep:`673`, Self type.
+* :pep:`675`, Arbitrary literal string type.
```

Comment:
This is generally PEP level items, I'm not sure I'd include a command-line option. The below is sorted by PEP number and grouped by normal / typing (as the [beta release announcement](https://discuss.python.org/t/python-3-11-0b2-is-now-available/16111) has been).

```suggestion
* :pep:`654`, Exception Groups and `except*`.
* :pep:`657`, Enhanced error locations in tracebacks.
* :pep:`680`, Support for parsing TOML in the standard library
* A ~1.25x speedup from 3.10

Typing features:
* :pep:`646`, Variadic generics.
* :pep:`655`, Marking individual TypedDict items as required or potentially-missing.
* :pep:`673`, Self type.
* :pep:`675`, Arbitrary literal string type.
* :pep:`681`, Dataclass transforms
```

A